### ISSUE TITLE
Toolbar and layer details layout updates

### ISSUE THREAD

**Issue by yvonnesjy** [NCEAS/metacatui#2345]
1. Change toolbar to overlay on top of map
2. Move layer details panel to the right of toolbar

**Comment by robyngit** [comment]
> Change toolbar to overlay on top of map

Note this is already the behaviour on small screens, so we could just apply those styles for all screen sizes

**Comment by yvonnesjy** [comment]
Yup it's definitely doable. My main concern right now is how to handle it in the 3-part view. It already doesn't have enough space in the current layout, and it definitely doesn't have enough space to have the toolbar and the details panel side-by-side.

**Comment by robyngit** [comment]
Maybe on screensizes smaller than some threshold, the layer list could just take up 100% of the screen? Just an idea.
